security/interceptor/security-interceptor-anonymous/src/main/java/org/codice/ddf/security/interceptor/AnonymousInterceptor.java

```diff
@@ -293,8 +297,21 @@ public void handleMessage(SoapMessage message) throws Fault {
                                     requireClientCert = evaluateExpression(xml, xpath);
 
                                 } else if (qName.equals(SP12Constants.SIGNED_SUPPORTING_TOKENS)) {
-                                    String xpath = "/SignedSupportingTokens/Policy/IssuedToken/RequestSecurityTokenTemplate/TokenType";
+                                    String xpath = "/SignedSupportingTokens/Policy/ExactlyOne/All/IssuedToken/RequestSecurityTokenTemplate/TokenType";
                                     tokenType = retrieveXmlValue(xml, xpath);
+                                    if (StringUtils.isEmpty(tokenType)) {
+                                        xpath = "/SignedSupportingTokens/Policy/IssuedToken/RequestSecurityTokenTemplate/TokenType";
+                                        tokenType = retrieveXmlValue(xml, xpath);
+                                    }
+                                    supportingTokenAssertion = qName;
+
+                                } else if (qName.equals(SP12Constants.SUPPORTING_TOKENS)) {
+                                    String xpath = "/SupportingTokens/Policy/ExactlyOne/All//IssuedToken/RequestSecurityTokenTemplate/TokenType";
```

Comment:
Double "/" after "All//"?

Looks like it should be the same as line 300 - maybe define these as constants?